shuup/gdpr/package.json

```diff
@@ -2,31 +2,22 @@
   "name": "shuup.gdpr",
   "version": "1.0.0",
   "description": "Enables General Data Protection Regulation (GDPR) in Shuup",
-  "main": "webpack.config.js",
+  "main": "",
   "scripts": {
-    "build": "npm i && webpack --env.dist",
-    "watch": "webpack --watch --env.dev --colors --progress"
+    "build:js": "./node_modules/.bin/parcel build -d static/ -o shuup-gdpr static_src/gdpr.js",
+    "build:styles": "./node_modules/.bin/parcel build --no-source-maps -d static/ -o shuup-gdpr static_src/gdpr.less",
+    "watch:js": "./node_modules/.bin/parcel watch --no-hmr -d static/ -o shuup-gdpr static_src/gdpr.js",
+    "watch:styles": "./node_modules/.bin/parcel watch --no-hmr --no-source-maps -d static/ -o shuup-gdpr static_src/gdpr.less",
```

Comment:
Don't separate the watch commands since you should be able to watch both js and styles simultaneously.

Reply:
I will probably try using https://github.com/mysticatea/npm-run-all or https://github.com/kimmobrunfeldt/concurrently

Reply:
I would also be a good idea to move the parcel commands to a separate scripts, since it's a PITA to have a kilometer long run script in the package.json.

Reply:
Hum, ok I will think in a some sort of script. I just want to make things easy to find. Currently to check how Admin is built, you have to open thousand gulp files. That is not so good, specially for new/noob users. We could have some script that uses Parcel API but I am not happy to create one just to replace this long line. A makefile will bee good as we could make things more organized but that will depend of having it installed so also not a good solution. So either a shell script or a node script. @suutari-ai thoughts?

Reply:
Maybe some variables to reduce duplicated values: http://www.marcusoft.net/2015/08/npm-scripting-configs-and-arguments.html#npm-configuration